pkg/servicelevel/types.go

```diff
@@ -251,6 +246,9 @@ type UserReference struct {
 	Name string `json:"name,omitempty"`
 }
 
+// EntityGUID - An encoded Entity GUID
+type EntityGUID string
```

Comment:
We should think about moving this out to a very basic module that has no dependancies to solve these circular import issues for common types and not re-defining the type.